core/src/main/java/org/apache/rocketmq/streams/core/function/supplier/MultiValueChangeSupplier.java

```diff
@@ -46,7 +50,16 @@ public MultiValueMapperProcessor(ValueMapperAction<T, ? extends Iterable<? exten
         public void process(T data) throws Throwable {
             Iterable<? extends VR> convert = valueMapperAction.convert(data);
 
+            if (convert == null) {
+                logger.warn("discard data:[{}]", convert);
+                return;
+            }
+
             for (VR item : convert) {
+                if (item == null) {
+                    logger.warn("discard data:[{}]", item);
```

Comment:
IMO, if it is null after convert, finish the process and return is nature, "discard data:[{}]" may cause misunderstanding.

Reply:
OK, so I should delete the `logger.warn` line and just return?

Reply:
Remove the log or modify to other sentence is ok.